data/themes/default/style.css

```diff
@@ -730,46 +830,46 @@ lmms--gui--TimeLineWidget {
 	qproperty-inactiveLoopColor: #3B424A;
 	qproperty-inactiveLoopBrush: #3B424A;
 	qproperty-inactiveLoopInnerColor: #3B424A;
-	qproperty-inactiveLoopHandleColor: rgba( 192, 192, 192, 100 );
+	qproperty-inactiveLoopHandleColor: rgba(192, 192, 192, 100);
 
 	/* Properties for the loop indicator rectangle in active state.
 	   See above for detailed description. */
 	qproperty-activeLoopColor: #21A14F;
 	qproperty-activeLoopBrush: #21A14F;
 	qproperty-activeLoopInnerColor: #21A14F;
-	qproperty-activeLoopHandleColor: rgba( 192, 192, 192, 200 );
+	qproperty-activeLoopHandleColor: rgba(192, 192, 192, 200);
 
 	/* Vertical padding for the loop indicator rectangle.
 	   A value of zero draws the rectangle at the full height of the widget. */
 	qproperty-loopRectangleVerticalPadding: 1;
 	/* Width of loop marker handles (when handle mode is active) */
 	qproperty-loopHandleWidth: 8;
 
-	qproperty-barLineColor: rgb( 192, 192, 192 );
-	qproperty-barNumberColor: rgb( 192, 192, 192 );
+	qproperty-barLineColor: rgb(192, 192, 192);
+	qproperty-barNumberColor: rgb(192, 192, 192);
 
 	/* Cursor hotspots for loop marker adjustment */
 	qproperty-mouseHotspotSelLeft: 0px 16px;
 	qproperty-mouseHotspotSelRight: 32px 16px;
 }
 
-lmms--gui--TrackContainerView QLabel
-{
+lmms--gui--TrackContainerView QLabel {
 	background: none;
 }
 
 /* Clips */
 
 /* common clip colors */
-lmms--gui--ClipView   {
-	qproperty-mutedColor: rgba(255,255,255,100);
+lmms--gui--ClipView {
+	qproperty-mutedColor: rgba(255, 255, 255, 100);
 	qproperty-mutedBackgroundColor: #373d48;
 	qproperty-selectedColor: #006B65;
 	qproperty-patternClipBackground: #373d48;
 	qproperty-textColor: #fff;
 	qproperty-textBackgroundColor: rgba(0, 0, 0, 75);
-	qproperty-textShadowColor: rgba(0,0,0,200);
-	qproperty-gradient: false; /* boolean property, set true to have a gradient */
+	qproperty-textShadowColor: rgba(0, 0, 0, 200);
+	qproperty-gradient: false;
+	/* boolean property, set true to have a gradient */
```

Comment:
Is it customary to put comments after lines in CSS?  I've never seen this before.

Reply:
No, and that comment doesn't even provide any useful information, lol. It looks like I had my autoformatter on, looking at the diff for `style.css` shows a ton of changes unrelated to what I was working on. Let me fix that.